packages/opentelemetry-core/src/trace/instrumentation/BasePlugin.ts

```diff
@@ -32,13 +55,77 @@ export abstract class BasePlugin<T> implements Plugin<T> {
     this._moduleExports = moduleExports;
     this._tracer = tracer;
     this._logger = logger;
+    if (config && config.basedir) this.basedir = config.basedir as string;
+    this._internalFilesExports = this._loadInternalFiles();
     return this.patch();
   }
 
   disable(): void {
     this.unpatch();
   }
 
+  private _loadInternalFiles(): ModuleExportsMapping {
+    let extraModules: ModuleExportsMapping = {};
+    if (this._internalFilesList) {
+      this._logger.debug('loadInternalFiles %o', this._internalFilesList);
+      Object.keys(this._internalFilesList).forEach(versionRange => {
```

Comment:
This arrow function seems kind of big. Could it be extracted as a private helper function that takes the `versionRange` and `extraModules`?

Reply:
Adjusted to use private helper function